analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/yarn-workspaces-expected-output.yml

```diff
@@ -0,0 +1,181 @@
+---
+project:
+  id: "Yarn::workspace:1.0.0"
+  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/package.json"
+  declared_licenses:
+  - "ISC"
+  declared_licenses_processed:
+    spdx_expression: "ISC"
+  vcs:
+    type: ""
+    url: ""
+    revision: ""
+    path: ""
+  vcs_processed:
+    type: "Git"
+    url: "ssh://git@github.com/heremaps/oss-review-toolkit.git"
+    revision: "04847ae49d9bb93810a2a83ce70c8a2aba71ceec"
```

Comment:
Use `<REPLACE_URL>` and `<REPLACE_REVISION>`.

Reply:
Done.

Reply:
Values are still hardcoded, also in the packages below.

Reply:
Done.